crates/modelardb_compression/src/models/pmc_mean.rs

```diff
@@ -162,77 +202,153 @@ mod tests {
 
     proptest! {
     #[test]
-    fn test_can_fit_one_value(value in ProptestValue::ANY) {
-        let error_bound_zero = ErrorBound::try_new(0.0).unwrap();
+    fn test_can_fit_one_value_with_absolute_error_bound(value in ProptestValue::ANY) {
```

Comment:
In the other tests where `ERROR_BOUND_ZERO` is used the test is named `with_absolute_error_bound_zero` or `with_relative_error_bound_zero`. Maybe be consistent with this naming scheme throughout the tests.

Reply:
Fixed.